recipes/lagraph/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# make sure CMake install goes in the right place
+export INSTALL="${PREFIX}"
+export CMAKE_OPTIONS="-DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_INSTALL_LIBDIR=lib"
+
+# make LAGraph
+pushd build
+cmake .. $CMAKE_OPTIONS
```

Comment:
Probably should do a release build

Reply:
What do you mean?  What is wrong and what might it be changed to?

Reply:
Alright, I think I got this sorted out by making it explicit with `-DCMAKE_BUILD_TYPE=Release`.  I'm not particularly competent at cmake, but this is what googling led me to.

Reply:
Yep - exactly!